eng/pipelines/templates/stages/vmr-verticals.yml

```diff
@@ -88,7 +88,7 @@ stages:
     - template: ../variables/vmr-build.yml
       parameters:
         vmrBranch: ${{ parameters.vmrBranch }}
-
+        buildSourceOnly: true
```

Comment:
I see that we are importing `variables/vmr-build.yml` in more places: https://github.com/search?q=repo%3Adotnet%2Fsdk%20variables%2Fvmr-build.yml&type=code. Should all of those get updated and pass buildSourceOnly in now?